docs/source/conf.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
```

Comment:
This can go away. ;-)
But unrelated to this PR.

Reply:
Fixed by ae03c23